core/server/api/settings.js

```diff
@@ -187,6 +138,7 @@ readSettingsResult = function (settingsModels) {
         apps = config.get('paths').availableApps,
         res;
 
+    // @TODO: why are availableThemes part of the settings cache?O_O
```

Comment:
It was done as a quick way to get a list of available themes to the settings panel in Ghost Admin a long, long time ago and never fixed. As far as I'm aware there still isn't a list endpoint for themes?